### PR TITLE
Update info on image fields for SMTP

### PR DESCRIPTION
**Pull request by aptalca**
## Proposed change

Updating info based on incoming PR to HA Core

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/93562

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — COMMENTED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — DISMISSED** [review]
Seems like something went wrong? There are now unrelated commits in this PR. Please git rebase it to clean it up.

../Frenck

**Comment by aptalca** [comment]
Yup, accidentally hit the `update branch` on github. Working on cleaning. Sorry about that.

File: source/_integrations/smtp.markdown
Comment (davet2001) [inline source/_integrations/smtp.markdown]: This seems to imply that if you add the html field then images and supporting html will be added inline automatically for you.

My understanding of the new behaviour makes me think that the following would be more accurate:

'If html is defined, this allows you to write your own html tags to define the location and properties of the images, in order to include them in the email'.

**Review by davet2001 — CHANGES_REQUESTED** [review]
Minor comment on wording.

File: source/_integrations/smtp.markdown
Comment (aptalca) [reply source/_integrations/smtp.markdown]: Good point. I changed the order and attempted to further clarify. Please let me know if this reads better.

File: source/_integrations/smtp.markdown
Comment (davet2001) [inline source/_integrations/smtp.markdown]: Possibly swap the word 'joined' for 'attached'.

**Review by davet2001 — CHANGES_REQUESTED** [review]
Minor reword suggestion but otherwise looks good now, thanks for the clarifications.

File: source/_integrations/smtp.markdown
Comment (aptalca) [reply source/_integrations/smtp.markdown]: Done

**Review by davet2001 — APPROVED** [review]
Looks good, thanks @aptalca 👍

**Comment by davet2001** [comment]
@aptalca please can you update from the base branch and mark as ready for review (non draft)?  Many thanks!

**Comment by aptalca** [comment]
@davet2001 Rebased and marked as ready for review

**Comment by davet2001** [comment]
Looks good.  From my point of view this can be merged once the parent PR is merged.

Thanks @aptalca👍

**Review by davet2001 — DISMISSED** [review]

**Comment by davet2001** [comment]
The parent PR is now merged.

File: source/_integrations/smtp.markdown
Comment (c0ffeeca7) [inline source/_integrations/smtp.markdown]:
```suggestion
The optional `html` field makes a custom text/HTML multi-part message, allowing total freedom for sending rich HTML emails by defining the HTML content. In them, if you need to include images, you can pass both arguments (`html` and `images`). The images will be attached with the basename of the images, so they can be included in the html page with `src="cid:image_name.ext"`.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @aptalca :+1:

**Pull request merged**
